app/popup.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Provider } from 'react-redux'
+import { BrowserRouter } from 'react-router-dom'
+
+import Popup from 'components/Popup'
+import store from 'store'
+
+store
```

Comment:
Why are you creating a new "react app" why not just use react-modal package or something like that in "index.js", what am I missing?

Reply:
The pop-up that is created to confirm transactions is not a modal, but a browser window. "index.js" just represents the chrome extension "window".